src/main/java/fr/zcraft/quartzlib/components/gui/InventoryGui.java

```diff
@@ -194,8 +193,12 @@ public void update() {
             // Reopening the inventory
             // FIXME This probably resets the player' mouse cursor position to the center of the GUI in 1.13.
             if (isOpen()) {
-                player.closeInventory();
+                RunTask.nextTick(() -> {
+                    player.closeInventory();
+                });
+
                 player.openInventory(inventory);
+
```

Comment:
Hi and thanks for this PR!

This will close the inventory after opening the new one. Are you sure this will work? Shouldn't both closing and opening be on the same tick?

Reply:
This should be better. I have delayed the reopening too. The first was in case that we try to close at the same tick the inventory was opened.